cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -466,14 +478,23 @@ func SetFields(dst interface{}, flags *pflag.FlagSet, prefix ...string) error {
 	return nil
 }
 
-var textUnmarshalerType = reflect.TypeOf((*encoding.TextUnmarshaler)(nil)).Elem()
+var (
+	errFieldRequired    = errors.DefineInvalidArgument("field_is_required", "field `{field}` is required")
+	textUnmarshalerType = reflect.TypeOf((*encoding.TextUnmarshaler)(nil)).Elem()
+)
 
 func setField(rv reflect.Value, path []string, v reflect.Value) error {
 	rt := rv.Type()
 	vt := v.Type()
 	props := proto.GetProperties(rt)
 	for _, prop := range props.Prop {
 		if prop.OrigName == path[0] {
+			if len(path) == 1 && v.String() == "null" {
+				if !prop.Required {
```

Comment:
This should not be necessary

Reply:
Indeed no fields are currently set as required (at least for end device updates that I checked). Don't know about the rest, plus I tried to make this safer for the future.

Reply:
Let's add this when we need it